ConcernsCaseWork/ConcernsCaseWork.Tests/Pages/Team/SelectColleaguesTests.cs

```diff
@@ -0,0 +1,171 @@
+﻿using ConcernsCaseWork.Models.Teams;
+using ConcernsCaseWork.Pages.Team;
+using ConcernsCaseWork.Security;
+using ConcernsCaseWork.Shared.Tests.Factory;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.AspNetCore.Mvc.Routing;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ITeamsModelService = ConcernsCaseWork.Services.Teams.ITeamsModelService;
+
+namespace ConcernsCaseWork.Tests.Pages.Team
+{
+	[Parallelizable(ParallelScope.All)]
+	public class SelectColleaguesTests
+	{
+		[Test]
+		public async Task WhenOnGetAsync_Return_Page()
+		{
+			// arrange
+			var testFixture = new TestFixture()
+				.WithUsersAvailableForSelection("user1", "user2", "Mr.Bean")
+				.WithPreviouslySelectedUsers("Mr.Bean");
+
+			var sut = testFixture.BuildSut(authenticatedPage: true);
+
+			// act
+			var pageResponse = await sut.OnGetAsync();
+
+			// assert
+			Assert.That(pageResponse, Is.Not.Null);
+			Assert.That(sut.SelectedColleagues, Is.Not.Null);
+			Assert.That(sut.Users, Is.Not.Null);
+			Assert.That(sut.Users.Length, Is.EqualTo(3));
```

Comment:
I'm being picky now. But is it possible to check that as well as having the right number of users, we have the right users? This test would pass if the same user were in there 3 times I think

Reply:
fair enough, added some asserts to check there is one of each